lua/nvim-treesitter/textobj.lua

```diff
@@ -0,0 +1,88 @@
+local api = vim.api
+local utils = require'nvim-treesitter.utils'
+local parsers = require'nvim-treesitter.parsers'
+local M = {}
+
+local function node_range_to_vim(node)
+  local select_range = [[
+  call cursor(%d, %d)
+  normal v
+  call cursor(%d, %d)
+  ]]
+  if node then
+    local start_row, start_col, end_row, end_col = node:range()
+
+    api.nvim_exec(string.format(select_range,
+      start_row+1, start_col+1,
+      end_row+1, end_col+1), false)
+  end
+end
```

Comment:
maybe 
```lua
if not node then return end
```
and then the rest of the code ?

Reply:
also `local exec_cmd = string.format(....)` and then `api.nvim_exec(exec_cmd, false)`